app/js/components/manageApps/ManageApps.jsx

```diff
@@ -691,7 +692,8 @@ export default class ManageApps extends React.Component {
             this.setState({
               appList: [],
               searchedAddons: [],
-              searchComplete: true
+              searchComplete: true,
+              isSearched: true
```

Comment:
I see that the value of `isSearch` is not changing but you keep setting it to state. Why is that?

Reply:
If you look through the codebase, you will see an instance where the `isSearch` value was changed

Reply:
Cool.